si/orange3-text.jaml

```diff
@@ -0,0 +1,5182 @@
+            lx_list: false
+        def `__relocate_file`:
+            basedir: false
+        def `update_preview`:
+            ', ': false
+            'Tokens: {n_tokens}\nTypes: {n_types}': Število členov: {n_tokens}\nŠtevilo različnih členov: {n_types}
+        def `workflowEnvChanged`:
+            basedir: false
+            sw_path: false
+            lx_path: false
+        def `migrate_settings`:
+            storedsettings: false
+            preprocessors: false
+            transformers: false
+            enabled: false
+            methods: false
+            checked: false
+            preprocess.transform: false
+            tokenizer: false
+            method: false
+            method_index: false
+            pattern: false
+            preprocess.tokenize: false
+            normalizer: false
+            snowball_language: false
+            udpipe_language: false
+            udpipe_tokenizer: false
+            preprocess.normalize: false
+            filters: false
+            recent_sw_files: false
+            recent_lexicon_files: false
+            language: false
+            stopwords_language: false
+            sw_path: false
+            sw_list: false
+            lx_path: false
+            lx_list: false
+            start: false
+            min_df: false
+            end: false
+            max_df: false
+            n_tokens: false
+            keep_n: false
+            preprocess.filter: false
+            ngrams_range: false
+            preprocess.ngrams: false
+            pos_tagger: false
+            tag.pos: false
+            rel_start: false
+            rel_end: false
+            abs_start: false
+            abs_end: false
+    __main__: false
+    deerwester: false
+widgets/owpubmed.py:
+    def `_i`:
+        icons: false
+    [^@]+@[^@]+\.[^@]+: false
+    class `OWPubmed`:
+        class `EmailCredentialsDialog`:
+            Pubmed Email: PubMed Email
+            class `Error`:
+                This email is invalid.: Ta email ni veljaven.
+            def `__init__`:
+                Email: true
+                email_input: false
+                Email:: true
+                OK: true
+        Pubmed: PubMed
+        Fetch data from Pubmed.: Pridobi podatke iz PubMed.
+        icons/Pubmed.svg: false
+        pubmed: pubmed
+        yyyy-MM-dd: false
+        %Y-%m-%d: false
+        class `Outputs`:
+            Corpus: Korpus
+        class `Warning`:
+            Please specify the keywords for this query.: Navedite ključne besede za to poizvedbo.
+        class `Error`:
+            'API error: {}.': API napaka: {}
+            Email not set. Pleas set it with the email button.: Email ni nastavljen. Prosimo, nastavite ga z gumbom Email.
+        def `__init__`:
+            Email: true
+            author: false
+            Author:: Avtor:
+            pub_date_from: false
+            pub_date_to: false
+            From:: Od:
+            to:: Do:
```

Comment:
```suggestion
            'to:': do:
```
It should be lowercase since it starts with `Od` and just continue some kind of sentence. Same as before with :. Ignore if not required.